var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -67,16 +82,19 @@ class Llvm(CMakePackage):
             description='CMake build type',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))
     variant('python', default=False, description="Install python bindings")
+    variant('flang', default='none',
+            description='Flang branch version, used by flang package',
+            values=('none', 'develop', '20180921', '20180612'))
+
     extends('python', when='+python')
 
     # Build dependency
     depends_on('cmake@3.4.3:', type='build')
-    depends_on('python@2.7:2.8', when='@:4.999 ~python', type='build')
-    depends_on('python', when='@5: ~python', type='build')
 
     # Universal dependency
-    depends_on('python@2.7:2.8', when='@:4.999+python')
-    depends_on('python', when='@5:+python')
+    depends_on('python@2.7:2.8', when='@:4.999')
+    depends_on('python')
+    depends_on('py-lit', type=('build', 'run'))
 
```

Comment:
What was the reason to revert the python dependency adjustments to always on vs `type='build'` when `~python`?

Reply:
Sorry, was not intentional.  Updated the pull request